include/swift/AST/DiagnosticsSema.def

```diff
@@ -2412,6 +2412,8 @@ ERROR(lazy_must_be_property,none,
       "lazy is only valid for members of a struct or class", ())
 ERROR(lazy_not_observable,none,
       "lazy properties must not have observers", ())
+ERROR(lazy_not_strong,none,
+      "lazy properties must not have reference attributes", ())
```

Comment:
We don't use the term "attributes" in diagnostics for context-sensitive keywords that act as modifiers, even though they're implemented that way. I think you should just write the name of the thing that can't be used, i.e. "lazy properties cannot be %0", taking the DeclAttribute as an argument.